students/kpersic/module_3/blog/scss/_icons.scss

```diff
@@ -0,0 +1,11 @@
+.fa-pencil {
+    margin-right: 9.6px;
```

Comment:
Gdje nađe 9.6px :D Decimalne vrijednosti u apsolutnim jedinicama poput px znaju uzrokovati vizualne bugove nekada. Postavi si pitanje: što je točno pola piksela? Kako to izgleda? Gdje se nalazi tih pola piksela u odnosu na neki piksel? Ne rješavaju svi preglednici taj problem na jednak način. Neki će zaokružiti prema dolje, neki prema gore, neki će uzeti median između dvije vrijednosti i naći mješavinu boja (zamutiti). Nisu sva rješenja vizualno atraktivna. Pripazi na to dugoročno.